### PR TITLE
Refactor message class

### PR DESCRIPTION
**Pull request by stevecalla**
## **_Pull Request (PR)_**
1. Is this a fix or a feature?
- [x] Refactor

2. Describe the feature / fix / refactor?
- [x] Adjust border radius to match comp
- [x] Refactor class to generate message (rather than in main.js)
- [x] Refactor class to eliminate duplicate messages

3. Describe the feature / fix / refactor?
- [x] See above.

4. Where should the reviewer start / test?
- [x] General testing to be sure app renders message

### CODE REVIEW

File: Message.js [hunk 1]
```diff
@@ -1,18 +1,68 @@
...
+
+  removeCurrentMessageFromAffirmationMantra() {
+    if (this.selectedMessageList === 'affirmations') {
+      var index = this.affirmations.indexOf(this.message);
+      this.affirmations.splice(index, 1); //WHY DOES THIS ALSO SPLICE AFFIRMATIONS ARRAY?
+    } else if (this.selectedMessageList === 'mantras') {
+      var index = this.mantras.indexOf(this.message);
+      this.mantras.splice(index, 1); //WHY DOES THIS ALSO SPLICE MANTRAS ARRAY?
+    }
+    console.log(this.affirmations.length, this.mantras.length);
```
Comment (PaigeVannelli) [inline Message.js:40]: Remember to remove console.logs!

File: Message.js [hunk 2]
```diff
@@ -1,18 +1,68 @@
...
+  }  
 
-    this.renderedAffirmations = this.renderedAffirmations + this.message;
+  resetMessageAffirmations() {
+    this.resetMessage = null; //only need to reset once...
+    if (this.affirmations.length === 0) {
```
Comment (PaigeVannelli) [inline Message.js:45]: You might be able to shorten this to a falsey value since you're checking if there's nothing in the array

File: Message.js [hunk 3]
```diff
@@ -1,18 +1,68 @@
...
+      }
+      this.renderedAffirmations = [];
+      this.setResetMessage('affirmations');
+    } 
   }
 
-  populateMatras() {
-    this.renderedMantras.push(this.message);
+  resetMessageMantras() {
+    if (this.mantras.length === 0) {
```
Comment (PaigeVannelli) [inline Message.js:55]: Same here with a falsey value

**Review by PaigeVannelli — COMMENTED** [review]
All in all it looks really good! Just a console.log and a few things that could be shortened

File: Message.js (hunk 1 above)
Comment (stevecalla) [reply Message.js:40]: will do before i submit the final.

File: Message.js (hunk 2 above)
Comment (stevecalla) [reply Message.js:45]: good catch... will take a look!

File: Message.js (hunk 3 above)
Comment (stevecalla) [reply Message.js:55]: yes. good catch!!
